### ISSUE TITLE
Reset masks are incorrect

### ISSUE THREAD

**Issue by techman-001** [pftbest/msp430#4]
Fabulous program Vadzim, MSP430 Forth users appreciate it greatly :)

Just one little problem so far, being the reset masks are incorrect, but everything else seems correct.

This is a list of ADC10 registers using the msp430g2553.svd Reset Mask values:-

\ ***** ADC10 ***** \
$48  constant  ADC10DTC0      \ Reset Mask:    255
$49  constant  ADC10DTC1      \ Reset Mask:    255
$4A  constant  ADC10AE0      \ Reset Mask:    255
$1B0  constant  ADC10CTL0      \ Reset Mask:    65535
$1B2  constant  ADC10CTL1      \ Reset Mask:    65535
$1B4  constant  ADC10MEM      \ Reset Mask:    65535
$1BC  constant  ADC10SA      \ Reset Mask:    65535

The Ti msp430x2xx user guide SLAU144F Page 606 showing inital reset states and they are different
--------------------------------------------------------------------------------------------------------------------------------------
Register                                Short Form  Register Type  Address  Initial State

ADC10 input enable register 0           ADC10AE0    Read/write     04Ah     Reset with POR
ADC10 control register 0                ADC10CTL0   Read/write     01B0h    Reset with POR
ADC10 control register 1                ADC10CTL1   Read/write     01B2h    Reset with POR
ADC10 memory                            ADC10MEM    Read           01B4h    Unchanged
ADC10 data transfer control register 0  ADC10DTC0   Read/write     048h     Reset with POR
ADC10 data transfer control register 1  ADC10DTC1   Read/write     049h     Reset with POR
ADC10 data transfer start address       ADC10SA     Read/write     01BCh    0200h with POR
-----------------------------------------------------------------------------------------------------------------------------------

Fetching the reset values on a actual msp430g2553 after a power reset
with Mecrisp 2.0.6 for MSP430G2553 by Matthias Koch, agrees with the SLAU144F pdf.
---------------------------------------------------------------------------------------------------------------------
reset-mask-test cr 
ADC10DAE0: 0000 
ADC10CTL0: 0000 
ADC10CTL1: 0000 
ADC10MEM: 0000 
ADC10DTC0: 0000 
ADC10DTC1: 0000 
ADC10SA: 0200

Cheers,
Terry

**Comment by pftbest** [comment]
Yes, I'm aware of this issue, but unfortunately I don't know a good way to fix it yet. Dslite files that I use doesn't provide reset values, and C headers that I also parse, don't have this information either. That's why I have to set them all to 0.

The only place I know that has a correct values is  in pdf files from TI. But I'm afraid that parsing a pdf documents is not so easy task.

**Comment by techman-001** [comment]
I wouldn't worry about it, it's very a very minor thing. I thought that they were reset values only because I've been parsing STM32F SVD's for so long (with SLST) and they have it. I searched thru all the msp430-gcc-support-files in your release and couldn't find any mention of reset values.

I could  extract MSP430x register reset values (with Forth) from a working chip into a text file file if you like, just let me know  ? I'd use your SVD to fetch them anyway, which by the way is just perfect, the data including the 'descriptions' is formatted perfectly for use in a programmers 'register reference file'.

Which is exactly what I have done.

Unlike  your msp430 the STM32F factory SVD's are all over the place, line feeds in the middle of 'definitions' along with long strings of spaces, it's a mess.

Cheers,
Terry
